Add GitHub profile

joelazwar opened the pull request:
Feature request #108 Adds display template for github profiles

joelazwar committed:
added github js and css template

joelazwar committed:
fixed styling issue and no bio logic

joelazwar committed:
cleaned up css

mdolr reviewed (changes requested):
Thank you, great pull request, could you just :
* Make the picture rounded (`border-radius:50%`)
* Display links below the repo / followers / following count
* Replace icon by text (e.g: "Twitter:", "Workplace:" (we will decide on an icon set later)

joelazwar committed:
changes as requested in #145, and polished a few styling settings

joelazwar committed:
styling for unavailable links

joelazwar committed:
minor typo in a display text

joelazwar commented:
@mdolr I made the changes as you requested and a few others

- Profile picture is now round
- Links are displayed below the repo/follower/following count, and i've aligned the counts to be centered as per container
- Replaced the icons for links with text for now, added some styling edits as well (e.g for unavailable links)

mdolr committed:
fix styling + show login when name is null + bump version number

mdolr committed:
Merge branch 'master' into add-github-profile

mdolr commented:
Merged :+1:
